.travis.yml

```diff
@@ -99,6 +99,9 @@ matrix:
     #   env: TRAVIS_PYTHON_VERSION=3.8
 
 before_install:
+    - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+        sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 762E3157;
```

Comment:
Should we have a plan for removing this? A key can be potentially compomised, and I'd assume that once Travis team fixes their images it will be included automatically.

Reply:
I added a check for key existence before key loading ;-)